src/main/java/module/TicTacToeService.java

```diff
@@ -0,0 +1,157 @@
+package main.java.module;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+public class TicTacToeService {
```

Comment:
The functions in this class are very big. Should be smaller.

Reply:
Can you please tell me which functions shall I try to make smaller, I have tried to implement only one functionality in each method, except the startGame method which controls the flow to each method

Reply:
The check methods